ams/server/src/main/java/com/netease/arctic/server/table/TableMetrics.java

```diff
@@ -0,0 +1,93 @@
+package com.netease.arctic.server.table;
+
+import com.netease.arctic.ams.api.metrics.*;
+import com.netease.arctic.server.metrics.MetricRegistry;
+import com.netease.arctic.server.optimizing.OptimizingStatus;
+import org.apache.iceberg.relocated.com.google.common.collect.ImmutableList;
+
+public class TableMetrics {
+  /** Table is no need optimizing. */
+  public static final String STATE_IDLE = "idle";
+
+  /** Table is need optimizing, but waiting for resource */
+  public static final String STATE_PENDING = "pending";
+
+  /** Table is doing optimizing process planing. */
+  public static final String STATE_PLANING = "planing";
+
+  /** Table is executing optimizing process */
+  public static final String STATE_EXECUTING = "executing";
+
+  /** All optimizing process task is done, and process is committing. */
+  public static final String STATE_COMMITTING = "committing";
+
+  private final ServerTableIdentifier identifier;
+
+  private String state = STATE_IDLE;
```

Comment:
Would it be more rigorous to use the OptimizingStatus enumeration directly?

Reply:
fixed.